up-core-api/uprotocol/core/usubscription/v3/usubscription.proto

```diff
@@ -183,34 +182,35 @@ message SubscriptionRequest {
 
 // Response message from the Subscribe() API
 message SubscriptionResponse {
-  // Current status of the subscription (state, code, etc...)
+  // Current status of the subscription
   SubscriptionStatus status = 1;
 
   // Any platform (uDevice) specific additional event delivered configuration
   // information for how the subscriber should consume the published events.
   EventDeliveryConfig config = 2;
 
   // Subscription topic passed to SubscriptionRequest
-  v1.UUri topic = 3;
+  uprotocol.v1.UUri topic = 3;
 }
 
 
 // Passed to the Unsubscribe() API, the message contains the topic to unsubscribe
 // to as well as the subscriber identification
 message UnsubscribeRequest {
   // Topic to be unsubscribed to
-  v1.UUri topic = 1;
+  uprotocol.v1.UUri topic = 1;
 
   // Subscriber identification
   SubscriberInfo subscriber = 2;
 }
 
+message UnsubscribeResponse {}
```

Comment:
Please document

Reply:
Hmm, what is there to document except for the obvious
> The (empty) response message representing the successful execution of the `Unsubscribe` operation.

This is exactly what we have defined in the Error Model specification. Do you think this is worth adding here again? Or am I missing something?

Reply:
yes the obvious :-)